src/net/net.zig

```diff
@@ -8,14 +8,10 @@ pub const SocketAddr = union(enum(u8)) {
 
     const Self = @This();
 
-    pub const UNSPECIFIED = Self{
-        .V4 = .{
-            .ip = Ipv4Addr{
-                .octets = [4]u8{ 0, 0, 0, 0 },
-            },
-            .port = 0,
-        },
-    };
+    pub const unspecified: SocketAddr = .{ .V4 = .{
```

Comment:
missed this one too

Reply:
c012dcfe881ad513bf546215be26fd5cfe70805d